src/components/molecules/Darkmode/DarkmodeContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import storage from '../../../lib/storage'
+import Darkmode from './Darkmode'
+import { compose, withStateHandlers, withPropsOnChange, withHandlers, lifecycle, pure} from 'recompose';
+
+export default compose(
+  pure,
+  withStateHandlers(
+    {
+      isActive: false,
+      isSupported: null
```

Comment:
@Joe-Pena With this method you can actually set a default which would eliminate the `componentWillMount` lifecycle. I'm a little confused on the `isSupported` state. Is it supposed to be a handler or a boolean value? It looks like you are simply using it to render the component or not. Are we ever calling the `isSupported` function?

```
withStateHandlers({
  isActive: storage.get('DarkMode') || false
},{
  toggleActive: ({ isActive }) => () => ({ isActive: !isActive })
})
```

Reply:
Great! Changed the isActive to your suggestion and moved the storage.set() to withPropsOnChange() as to get rid of the entire lifecycle method.

Will implement a browser support check later on, as only IE and Opera Mini do not support the CSS filter property, so anything related to isSupported has been removed.